File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.py
def solution(numLog):
  answer = ''
  for idx in range(1, len(numLog)):
    value = numLog[idx] - numLog[idx-1]
    if value == 1:
      answer += 'w'
    elif value == -1:
      answer += 's'
    elif value == 10:
      answer += 'd'
    elif value == -10:
      answer += 'a'
  
  return answer

File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.js
function solution(numLog) {
  var answer = '';
  
  const order = {
    '1': 'w',
    '-1': 's',
    '10': 'd',
    '-10': 'a'
  };

  return numLog.slice(1).map((v, i) => order[v - numLog[i]]).join('')
}